File: programmers/2020_09/day30.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        
        // 同じ数字なら、aとかbをリーたんする。
        if (a == b) {
            return answer = a;
        } else if (a < -10000000 || a > 10000000) {
            //　aのサイズチェック
            return answer;
        } else if (b < -10000000 || b > 10000000) {
            //　bのサイズチェック
            return answer;
        } else {
            // aは必ず小さい数字に設定するため、aとbのサイズをチェックする。（bが小さい場合aのサイズに移動する）
            if (a > b) {
                int tmp = a;
                a = b;
                b = tmp;
            }
            //　初期値をa、aからbまで繰り返しにする。
            for (int i = a; i <= b; i++) {
                answer = answer + i;
            }
        }
        return answer;
    }
}

File: programmers/2020_11/day30.py
def solution(arr, divisor):
    answer = []
    # 배열길이 구하는 방법 len()
    arrLen = len(arr)
    # 제한사항
    if (1 > arrLen) :
        return answer
    
    for i in range(0, arrLen) :
        if (arr[i] % divisor == 0) :
            # arr[i]에 divisor를 나눠서 나머지가 0인경우 배열에 추가
            answer.append(arr[i])
    
    answerLen = len(answer)
    # answer배열의 길이가 0인경우 -1을 반환한다.
    if (0 == answerLen) :
        return [-1]
    
    # 오름차순 정렬
    answer.sort()
    # 참고 reverse()를 이용하면 현재 배열을 반대로 뒤집는다.
            
    return answer
